freqtrade/data/entryexitanalysis.py

```diff
@@ -120,7 +120,7 @@ def _do_group_table_output(bigdf, glist):
 
         else:
             agg_mask = {'profit_abs': ['count', 'sum', 'median', 'mean'],
-                        'profit_ratio': ['sum', 'median', 'mean']}
+                        'profit_ratio': ['median', 'mean', 'sum']}
             agg_cols = ['num_buys', 'profit_abs_sum', 'profit_abs_median',
                         'profit_abs_mean', 'median_profit_pct', 'mean_profit_pct',
                         'total_profit_pct']
```

Comment:
@froggleston 
i was doubting the same at first - but:
if you compare the changed fields (`agg_mask`) with `agg_cols` - you'll notice that the sequence is wrong.
(total_profit_pct != profit_ratio.mean) ...

Reply:
Oh great spot @Undertoned ! LGTM.